README.md

```diff
@@ -1,5 +1,97 @@
 # e-commerce-app
 
-## My first Pull!
+This is a responsive app of an online clothing store.
```

Comment:
Perhaps something more general like so:

This an e-commerce web-site.
A user can buy clothes but looking at images and filling a basket with products.... etc.

IMO - you should include an intro as well on the motives of the project.
i.e.
This application is part of a learning project and will include the following features:
1. Responsive web design.
2. Payment functionality.
3. Data base for storing and retrieving user and products data.
4. etc..